common/utils.go

```diff
@@ -144,3 +145,23 @@ func GetBridgeNetDev(bridgeName string) ([]NetDev, error) {
 		return link.Attrs().Name == bridgeName
 	})
 }
+
+// Configure the ARP cache parameters for the given interface.  This
+// makes containers react more quickly to a change in the MAC address
+// associated with an IP address.
+func ConfigureArpCache(ifname string) error {
```

Comment:
Why would we not want it to error? We expect the file to be there as part of procfs - if it's not, something is wrong